sublime-package.json

```diff
@@ -937,16 +432,17 @@
               "enum": [
                 "off",
                 "basic",
+                "standard",
                 "strict"
               ],
               "title": "Specifies the default rule set to use for type checking",
-              "default": "basic"
+              "default": "standard"
```

Comment:
pyright has recently changed the default value of `typeCheckingMode` from `basic` to `standard`. We have to decide whether we want to follow that in our settings or not. `standard` causes triggers some errors in LSP code base at least.

Reply:
I think it would make sense to switch to `standard` since only then the default values defined in schema should actually match the reality. The issue with the LSP code base was only related to a single diagnostic `reportIncompatibleMethodOverride`.